src/feature-lifecycle/stabilization.md

```diff
@@ -1,22 +1,43 @@
+
+## Stabilization Report
+
+Once a feature is ready for stabilization the first step of the FCP process is writing a stabilization report. Stabilization reports consist of three primary sections, a implementation history, an API summary, and an experience report.
+
+The **Implementation History** section should summarize the initial discussion during the implementation PR, every change that has been made to the feature since the initial implementation, all issues that were raised during the lifetime of the feature, and how they were resolved.
+
+The **API Summary** section should include a precise description of what APIs are being introduced to the standard libraries. The libs team maintains a tool for this called [`cargo unstable-api`](https://github.com/rust-lang/libs-team/tree/main/tools/unstable-api) that can be used to generate these API summaries in some cases. *Note* the current implementation of this tool is fragile and does not work in all cases. We hope to have a more permanent version of this tool in the future that is built ontop of either rustdoc or rustc's own APIs.
+
+The **Experience Report** section should include links to projects that use the feature being stabilized, or when applicable, projects that use crates that export the same API and describe the experience of using the feature in question.
+
+You can see examples of stabilization reports in tracking issues [#88581](https://github.com/rust-lang/rust/issues/88581#issuecomment-1054642118) and [#29553](https://github.com/rust-lang/rust/issues/29553).
 
 ## Before writing a PR to stabilize a feature
 
 Check to see if a FCP has completed first. If not, either ping `@rust-lang/libs` or leave a comment asking about the status of the feature.
 
 This will save you from opening a stabilization PR and having it need regular rebasing while the FCP process runs its course.
 
-## Writing a stabilization PR
+## Partial Stabilizations
 
-- Replace any `#[unstable]` attributes for the given feature with stable ones. The value of the `since` field is usually the current `nightly` version.
-- Remove any `#![feature()]` attributes that were previously required.
-- Submit a PR with a stabilization report.
+When you only wish to stabilize a subset of an existing feature you should skip creating a new tracking issue and instead create a partial stabilization + tracking PR for the subset of the feature being stabilized. When you're splitting up the feature, you should always introduce a new feature name for the portion being stabilized and leave the existing feature for the portions left unstable, that way existing users of that feature are not left in a broken state.
```

Comment:
I'm not sure it always makes sense to pick a new name for the stable part and keep the name for the unstable part. If the main parts of a feature get stabilized, it might make more sense for the more niche parts to get their own feature name(s) instead.

Reply:
This was added because of [this comment](https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/First.20time.20going.20through.20FCP/near/274262666) by @scottmcm. I'm having trouble imaging a scenario where the name of the portion being stabilized being the same as the original feature name would matter more than the than not breaking the code of the people who were using that feature name to enable the portions of the feature left unstable.

Reply:
If we stabilize the majority of some feature except for a small less commonly used part, I think it's better to keep the original feature name and move the smaller part to a new name. That way, most users will just get a warning saying that the `#![feature(..)]` they enable has been stabilized and they can just remove it. Those who do use the small part that has not been stabilized yet can then also consider updating their code to avoid the still unstable part, since they might now be able to do what they want with the big part of the feature that has been stabilized.

For example, imagine we'd stabilize `Saturating` except for `Saturating::pow`. Then we should probably keep the same feature name for the stabilization and use a new feature name for the still unstable `pow` method.

Reply:
Is the issue here that people don't get a warning if they enable a feature and never use it? So in the case where they enabled the old feature name and we stabilize all the APIs that they used under a new feature name they would be left in the dark and continue to use the old feature and stay on nightly much longer than needed.